source/com.squareup.okhttp3/okhttp-jvm/Additions/Additions.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+
+namespace Square.OkHttp3.Internal.Connection
+{
```

Comment:
If `Square.**.Internal` namespaces and packages are removed, remove this entire file.